src/components/avatar/avatar.ts

```diff
@@ -0,0 +1,13 @@
+import template from './avatar.hbs';
+import './avatar.scss';
+import Block from '../../modules/Block';
+
+class Avatar extends Block {
```

Comment:
```suggestion
export class Avatar extends Block {
```
Часто удобно пользоваться именованными импортами:
- TS явно подсвечивает, что вы потом можете экспортировать из файла;
- удобно организовываться ситуации, когда из модуля экспортируется несколько компонентов;